test/test-utils.sh

```diff
@@ -343,8 +345,15 @@ function make_random_string() {
     else
         END_POS=8
     fi
-    RANDOM_STR=`cat /dev/urandom | base64 | sed 's#[/|+]##g' | head -1 | cut -c 1-${END_POS}`
-    echo "${RANDOM_STR}"
+
+    # [NOTE]
+    # If character substitution occurs, it will be less than the specified
+    # number of characters, but the value is only used for the file name
+    # now, so it is acceptable.
+    #
+    ${BASE64_BIN} --wrap=0 < /dev/urandom | head -c ${END_POS} | sed -e 's#[/|+]##g'
```

Comment:
Does this return too few characters?  You should reverse `head` and `sed`:

```sh
${BASE64_BIN} --wrap=0 < /dev/urandom | sed -e 's#[/|+]##g' | head -c ${END_POS}
```

But it is probably better to replace `sed` with `tr` which likely handles long lines better:

```sh
${BASE64_BIN} --wrap=0 < /dev/urandom | tr -d /+ | head -c ${END_POS}
```

Reply:
Thank you, it seems better to put tr in between as you point.
(Unlike sed, I comfirmed that tr is not blocked on macOS.)